packages/core/lib/commands/test/run.js

```diff
@@ -107,6 +107,22 @@ module.exports = async function (options) {
     }
   };
 
+  function sanitizeGanacheOptions(ganacheOptions) {
+    if (ganacheOptions.network_id === "*") {
+      ganacheOptions.network_id = 4447;
```

Comment:
I would like to suggest that this function should probably be pure and not mutate the input; it would probably be better to clone with spreads rather than to directly mutate like this.

Granted, in this case it's harmless because it's only used in one place, but unnecessary mutation makes code harder to understand, so it's good to avoid it when reasonable.

Reply:
Thanks @haltman-at for your suggestion. I will make the code better.